openlibrary/templates/home/stats.html

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
```suggestion
```
Does removing this blank line enable the Travis CI tests to pass again?  This is touchy syntax in web.py webpy/webpy#630 because `with` is a Python keyword so defining a function named `with` would be a Python syntax error.